comid-code-points.cddl

```diff
@@ -6,6 +6,12 @@ comid.entity = 3
 comid.linked-tags = 4
 comid.claims = 5
 
+; ref-instance-claim-map
+comid.ref-claim-value = 1
+
+; endors-instance-claim-map
+comid.endors-claim-value = 1
```

Comment:
Both would clash with `comid.mac-addr`.  The first usable code-point is 6 (please check it).

Reply:
Have verified and new available code point is 6 so started from their.